src/Client/Connection.cpp

```diff
@@ -335,6 +337,11 @@ void Connection::sendHello()
         performHandshakeForSSHAuth();
     }
 #endif
+    else if (!jwt.empty())
+    {
+        writeStringBinary(EncodedUserInfo::JWT_AUTHENTICAION_MARKER, *out);
+        writeStringBinary(jwt, *out);
```

Comment:
Here `jwt` is sent but I couldn't find in this PR where it's received and processed. Is this PR finished?

Reply:
This PR is only for handling JWT in client.

Reply:
But what is the point of sending jwt from clickhouse-client if clickhouse-server can't handle it?